src/NHibernate/Cfg/MappingSchema/Hbm.generated.cs

```diff
@@ -4294,14 +4294,18 @@ public enum HbmFlushMode {
         /// <remarks/>
         [System.Xml.Serialization.XmlEnumAttribute("auto")]
         Auto,
-        
+
         /// <remarks/>
-        [System.Xml.Serialization.XmlEnumAttribute("never")]
-        Never,
-        
+        [System.Xml.Serialization.XmlEnumAttribute("manual")]
+        Manual,
+
         /// <remarks/>
         [System.Xml.Serialization.XmlEnumAttribute("always")]
         Always,
+
+        /// <remarks/>
+        [System.Xml.Serialization.XmlEnumAttribute("never")]
+        Never,
```

Comment:
Semantically cleaner, but not sure the generator from xsd would do that, it has no mean to know it should (or I do not know how to tell it in xsd).